scripts/serie_update.py

```diff
@@ -525,6 +526,38 @@ def update_cmakelist(self):
                     )
         os_cmd(("dos2unix", str(cmakelists_path)))
 
+    def generate_assert_file(self):
+        """Remove stm32_assert_template.h file and create stm32_assert.h file"""
+        # remove stm32_assert_template.h
+        stm32_assert_template = (
+            self.stm32cube_temp_serie
+            / "drivers"
+            / "include"
+            / "stm32_assert_template.h"
+        )
+        if stm32_assert_template.exists():
+            os.remove(stm32_assert_template)
+
+        # create stm32_assert.h from Jinja2 template
+        # Create the jinja2 environment.
+        templates_dir = self.zephyr_hal_stm32_path / "scripts"
+        j2_env = Environment(
+            loader=FileSystemLoader(str(templates_dir)),
+            trim_blocks=True,
+            lstrip_blocks=True,
+        )
+        stm32_assert_j2_template = j2_env.get_template(
+            "stm32_assert_template.txt"
```

Comment:
Shouldn't that be a full path?

Reply:
It is not needed because path is provided tanks to `Environment()`:
```
        j2_env = Environment(
            loader=FileSystemLoader(str(templates_dir)),
            trim_blocks=True,
            lstrip_blocks=True,
        )
```